docs/user/admin/settings.rst

```diff
@@ -16,36 +16,6 @@ Introduction
 
 When OpenSearch bootstraps, SQL plugin will register a few settings in OpenSearch cluster settings. Most of the settings are able to change dynamically so you can control the behavior of SQL plugin without need to bounce your cluster. You can update the settings by sending requests to either ``_cluster/settings`` or ``_plugins/_query/settings`` endpoint, though the examples are sending to the latter.
 
```

Comment:
do we need to list the removed settings in breaking change?

Reply:
> do we need to list the removed settings in breaking change?

As long as we include this change in the release notes, I think it is less confusing to remove them from the documents.